src/main/java/desktop/pages/HomePage.java

```diff
@@ -2,42 +2,44 @@
 
 import abstractclasses.page.AbstractPage;
 import desktop.fragments.NavigationBarFragment;
+import driver.SingletonDriver;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
+import static constants.Constants.HOME_PAGE_URL;
+
 public class HomePage extends AbstractPage {
-    @FindBy(xpath = "//input[@name=\"searchTerm\"]")
+    NavigationBarFragment navigationBar = new NavigationBarFragment(driver);
+    @FindBy(xpath = "//input[@name='searchTerm']")
     private WebElement searchBarInput;
-
-    @FindBy(xpath = "//button[@class=\"header-search-btn\"]")
+    @FindBy(xpath = "//button[@class='header-search-btn']")
     private WebElement searchButton;
-
-    @FindBy(xpath = "//div[@class=\"secondary-header\"]")
+    @FindBy(xpath = "//div[@class='secondary-header']")
     private WebElement navigationCategories;
-
     @FindBy(xpath = "//*[@class='book-item']")
     private WebElement searchResults;
 
-    public HomePage(WebDriver driver){
+    public HomePage(WebDriver driver) {
         super(driver);
+        SingletonDriver.getInstance().get(HOME_PAGE_URL);
     }
-    NavigationBarFragment navigationBar = new NavigationBarFragment(driver);
 
-    public void enterSearchTerm(){
-    searchBarInput.sendKeys("Thinking");
+    public void enterSearchTerm() {
```

Comment:
Please accept search term as a parameter (you can also just change method's name to enterThinkingSearchTerm, but I'd suggest not to do this :) )

Reply:
fixed

Reply:
there is no "resolve conversation" button here, so I will just put a "thumbs up" emotion or a comment to all other "resolved" threads